src/py/flwr/client/clientapp/clientappio_servicer.py

```diff
@@ -68,6 +70,7 @@ class ClientAppIoServicer(clientappio_pb2_grpc.ClientAppIoServicer):
     def __init__(self) -> None:
         self.clientapp_input: Optional[ClientAppIoInputs] = None
         self.clientapp_output: Optional[ClientAppIoOutputs] = None
+        self.token: Optional[int] = None
```

Comment:
This is not needed, the token is already part of `ClientAppIoInputs`. We should, however, track whether the token has been delivered or not.